coremltools/converters/mil/mil/ops/defs/random.py

```diff
@@ -36,37 +36,38 @@ def type_inference(self):
 """
 
 
-@register_op(
-    doc_str=r"""
-Returns a tensor with specified shape with random values from a Bernoulli distribution.
-
-.. math::
-
-    f(k) = \begin{cases}1-p  &\text{if } k = 0\\
-                        p    &\text{if } k = 1\end{cases}
-
-for :math:`k` in :math:`\{0, 1\}`.
-
-Parameters
-----------
-shape: <K, i32>, required
-    Target output tensor shape.
-    K is the rank of the output tensor. shape[k] > 0 for k = 0,..., K-1.
-prob: const<f32>, optional
-    The probability of sampling 1. Defaults to 0.5.
-seed: const<i32>, optional
-    Seed to create a reproducible sequence of values across multiple invokes.
-
-Returns
--------
-<*, T>, a tensor of given target output shape filled with random values.
-
-See Also
---------
-random_categorical, random_normal, random_uniform
-"""
-)
+@register_op(doc_str="TODO")
 class random_bernoulli(RandomDistribution):
+    """
+    Returns a tensor with the specified shape, with random values from a Bernoulli
+    distribution.
+    
+    .. math::
+       f(k) = \\begin{cases}1-p  &\\text{if } k = 0\\\\p &\\text{if } k = 1\\end{cases}
```

Comment:
Why change the formatting of the equation? Looks okay in preview though, I preferred the way it was written before though, since that one is easier to read in latex as well.

Reply:
My mistake. I had to move the docstrings in this file, so that numpydoc would pick them up, and I forgot the `r` of `"""r`; this equation was also causing a "\b error" in the JSON when running the publishing script. 

After reading about escaping the backslashes (in MathTex docs), I doubled-up the backslashes in the equation in order to make the equation look like the result in the Online LaTeX Equation Editor. While it works in the preview, the docstring itself became harder to read.

To fix, I am reverting the equations and using `"""r` in this file.